resources/lib/addon.py

```diff
@@ -105,6 +105,7 @@ def favorites_refresh():
 @plugin.route('/programs')
 @plugin.route('/programs/<program>')
 @plugin.route('/programs/<program>/<season>')
+@plugin.route('/programs/<program>/<season>/')
```

Comment:
This means any constructed URL within the add-on will have a trailing slash.
Is this what we want?

Reply:
I think if we swap the position of both, this is no longer an issue.

Reply:
I added this route because I found out Kodi built-in `ActivateWindow` doesn't handle urls without a trailing slash right. I didn't found another way to fix this.

Reply:
Because now it creates routes with trailing slashes internally:
```
» Season 2019 → /programs/pano/2019/
» Season 2018 → /programs/pano/2018/
» Season 2017 → /programs/pano/2017/
» Season 2016 → /programs/pano/2016/
```

Reply:
If I change the order I get `No route to path` for all plugins url that are handled by `ActivateWindow`.

Reply:
Looking into this deeper, the problem seems to only exist when the season consists of numbers (e.g. `2016`), but not if it is a string (e.g. `allseasons`).

Reply:
I reported this `ActivateWindow()` issue upstream at:: https://github.com/xbmc/xbmc/issues/16516